lib/config/executor/container_vm.go

```diff
@@ -44,6 +44,21 @@ type Common struct {
 	Notes string `vic:"0.1" scope:"hidden" key:"notes"`
 }
 
+// Common data (specifically for a containerVM) between managed entities, across execution environments.
+type Common2 struct {
```

Comment:
@sgairo We want to persist the `name` in Common while the containeVM is still running, so we need to set the property of `name` to `hidden`. In order not to mess up with the original `Common` which might require `read-only` name by other objects, we create `common2` instead.